_dev/.stylelintrc

```diff
@@ -4,7 +4,8 @@
     "stylelint-config-recommended-scss"
   ],
   "ignoreFiles": [
-    "../packages/**/tailwind.out.scss"
+    "../**/tailwind.out.scss",
```

Comment:
TIL there's a tailwind out SCSS file?? Wonder if that's a relic from something else.